### PR TITLE
Artist matching should now be a little bit more accurate.

### PR DESCRIPTION
**Pull request by jamesjarvis**
Because ABBA !== Abbas Hamad

### CODE REVIEW

File: src/backend/genius/artist.js [hunk 1]
```diff
@@ -4,17 +4,23 @@ const appleMusicApi = require('../appleMusicApi');
...
 
-async function findMatch(hits) {
+async function findMatch(hits, name) {
   if(hits.length === 0) {
     return null;
   }
 
   for (const hit of hits) {
-    if (hit.type === "artist") {
+    if (hit.type === "artist" && hit.result.name.toLowerCase() == name.toLowerCase()) {
```
Comment (raphaelvigee) [inline src/backend/genius/artist.js]: use `stringComparator` from `song.js`

File: src/backend/genius/artist.js [hunk 2]
```diff
@@ -76,4 +82,4 @@ module.exports = {
       return utils.generateError(500, e);
     }
   }
-};
+};
```
Comment (raphaelvigee) [inline src/backend/genius/artist.js]: \n

File: src/backend/genius/artist.js (hunk 1 above)
Comment (jamesjarvis) [reply src/backend/genius/artist.js]: ooooh hadn't event seen that, sweet

**Comment by jamesjarvis** [comment]
I dunno why this is saying the changes havent been addressed...

**Pull request merged**